src/Components/WebAssembly/WebAssembly.Authentication/src/Options/OidcProviderOptions.cs

```diff
@@ -57,5 +57,10 @@ public class OidcProviderOptions
         /// </summary>
         [JsonPropertyName("response_mode")]
         public string ResponseMode { get; set; }
+
+        /// <summary>
+        /// Gets or sets the extra query parameters to use on the authorization flow. 
+        /// </summary>
+        public IDictionary<string, string> ExtraQueryParams { get; } = new Dictionary<string, string>();
```

Comment:
Change the name to AdditionalProviderParameters or something more descriptive. We normally don't abbreviate these things.

Reply:
Do you have any suggestion for the json prop name as well? after rename default would be `additionalProviderParameters` in this case @javiercn